src/openPr.js

```diff
@@ -24,7 +25,21 @@ const addArtifact = async (inputs, releaseId) => {
   return artifact
 }
 
-const createDraftRelease = async (inputs, newVersion) => {
+const getReleaseNotes = async (inputs, newVersion) => {
+  const latestRelease = await fetchLatestRelease(inputs)
+  if (!latestRelease) {
+    return null
```

Comment:
let's not return nulls explicitly, it's not idiomatic. 

also, let's try to stick to a convention with functions that either return what they are meant to, or falsy. prefix their names with "try", e.g. tryGetReleaseNotes. this is to signify that either it does what it should, or it returns something (usually a falsy value) without throwing. the same name without the "try" prefix would instead throw an error if it can't do what we're asking it to do